OpenDreamShared/Compiler/DM/DMParser.cs

```diff
@@ -109,7 +109,29 @@ public partial class DMParser : Parser<Token> {
                             Whitespace();
 
                             DMASTExpression value;
-                            if (Check(TokenType.DM_Equals)) {
+                            if (Check(TokenType.DM_LeftBracket)) //TODO: Multidimensional lists
+                            {
+                                //Type information
+                                if (varPath.FindElement("list") != 0)
```

Comment:
`var/list[]` - not typed by your pathstring stuff?

Reply:
lists named `list` already don't work for other reasons